scss/mixins/_utilities.scss

```
@@ -9,6 +9,8 @@
   }
 
   @each $value in $values {
```

I guess you can just use this here: 
```suggestion
  @each $key, $value in $values {
```